kv/local_sender_test.go

```diff
@@ -43,6 +44,36 @@ func TestLocalSenderAddStore(t *testing.T) {
 	}
 }
 
+func TestLocalSenderRemoveStore(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	ls := NewLocalSender()
+
+	store := storage.Store{}
```

Comment:
Zero is not a valid store ID. You should give this store a real ID (like you did below), and in between `AddStore` and `RemoveStore` verify that `HasStore` returns true for that ID.